woocommerce/woocommerce-android
# Check for shipping label support

Part of: #12444 


### Description
This PR introduces the `ShippingLabelSupport` enum to give us more control over the shipping label flow. With the changes introduced here, we can check if an order supports purchasing shipping labels and whether we should use the new or legacy flow.

### Testing information
TC1
1. Install & Activate the WCS&T plugin on your store
2. Check that the purchase shipping label flow keeps working as expected

TC2
1. Install & Activate the WCS plugin on your store
2. Check that the purchase shipping label flow keeps working as expected

### The tests that have been performed
1. Tested that the purchase shipping label flow keeps working as expected using the WCS&T and the WCS plugins


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
